src/react/util.ts

```diff
@@ -1,11 +0,0 @@
-import { QueryResult, loading, success } from '../QueryResult';
```

Comment:
I renamed this file into `Semigroup` since its content is now also exported publicly (see `react/index.ts`)